deploy/kube-config/riemann/riemann-service.json

```diff
@@ -0,0 +1,36 @@
+{
+  "apiVersion": "v1beta3",
+  "kind": "Service",
+  "metadata": {
+    "labels": {
+      "kubernetes.io/cluster-service": "true",
+      "name": "riemann-heapster"
+    },
+    "name": "riemann-heapster"
+  },
+  "spec": {
```

Comment:
It's a good idea, but I'm tempted to defer. We have run with riemann-dash in the past, and we're not currently doing so; it's a useful tool for exploration and certain visualizations. In practice, we stream our metrics from Riemann to Grafana dashboards.